{"name": "Проверка логина с неверными данными", "status": "passed", "description": "\n                                1. Генерим случайные данные;\n                                3. Оправляем запрос на логин;\n                                3. Проверяем ответ;\n                                ", "start": 1718223853156, "stop": 1718223853608, "uuid": "c1323858-9830-40e3-aa16-0b468da15afb", "historyId": "607399e3c691e986195ddf64bb28a636", "testCaseId": "607399e3c691e986195ddf64bb28a636", "fullName": "tests.test_login_user.TestCreateUser#test_login_incorrect_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "otemorygame"}, {"name": "thread", "value": "22988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}